src/Attachment/index.ts

```diff
@@ -317,9 +317,9 @@ export class ResponsiveAttachment implements ResponsiveAttachmentContract {
   }
 
   protected async enhanceFile(): Promise<ImageInfo> {
-    // Read the image as a buffer using `Drive.get()`
-    const originalFileBuffer = this.buffer ?? (await this.getDisk().get(this.relativePath!))
-
+    // Read the image as a buffer using `Drive.get()`, normalizing the path
+    const originalFileBuffer =
+      this.buffer ?? (await this.getDisk().get(this.relativePath!.replace(/\\/g, '/')))
```

Comment:
Please replace the modified lines with:

```ts
    // Read the image as a buffer using `Drive.get()`
    // Make sure path is normalised
    const originalFileBuffer =
      this.buffer ?? (await this.getDisk().get(normalize(this.relativePath!)))
```

Import `normalize` from `path` at the top-level imports.